View/CsvView.php

```diff
@@ -293,8 +293,15 @@ protected function _generateRow($row = null) {
 		$newline = $this->viewVars['_newline'];
 
 		$row = str_replace(array("\r\n", "\n", "\r"), $newline, $row);
-		if (fputcsv($fp, $row, $delimiter, $enclosure) === false) {
-			return false;
+		if ($enclosure == '') {
```

Comment:
You are right. I am not 100% familiar with the GitHub workflow. What should I do next? Fix it on my branch, commit to master and then create a new pull request? Or is it possible to add a new commit to this pull request? Thank you in advance!